pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -89,13 +89,36 @@
   oceans_behind_the_scenes_2: 'Levels 6-8 use a <a href="https://en.wikipedia.org/wiki/Support-vector_machine">Support-Vector Machine</a> (SVM). We look at each component of the fish (such as eyes, mouth, body) and assemble all of the metadata for the components (such as number of teeth, body shape) into a vector of numbers for each fish. We use these vectors to train the SVM. Based on the training data, the SVM separates the "space" of all possible fish into two parts, which correspond to the classes we are trying to learn (such as "blue" or "not blue").'
   oceans_behind_the_scenes_2_markdown: 'Levels 6-8 use a [Support-Vector Machine](https://en.wikipedia.org/wiki/Support-vector_machine) (SVM). We look at each component of the fish (such as eyes, mouth, body) and assemble all of the metadata for the components (such as number of teeth, body shape) into a vector of numbers for each fish. We use these vectors to train the SVM. Based on the training data, the SVM separates the "space" of all possible fish into two parts, which correspond to the classes we are trying to learn (such as "blue" or "not blue").'
 
-  heading_additional_resources: 'Additional resources'
+  social_hoc2019_oceans_title: 'AI for Oceans #CSforGood'
```

Comment:
I'm curious, since the new page is on code.org, should the translations be added somewhere in `pegasus/sites.v3/code.org/i18n` instead?

Reply:
It looks like this pattern is pretty widely used already, so I am going to say it is fine for this PR, but I am also going to follow up with the platform team to see if we should be defining these strings differently.

Reply:
Ideally yes, this has already been brought up to the platform team to try and get a better system for code.org translations set up, but in the meantime we've been using the hourofcode file. There's some more info  [here](https://codedotorg.slack.com/archives/C0T0PNTM3/p1670886528814599?thread_ts=1670877588.726089&cid=C0T0PNTM3) and [here](https://docs.google.com/document/d/1pWGWKyWspFIAHC1-E6xKCW_9MOLh7914Pe3Qfe6Yct0/edit?usp=sharing).